apis/farms/src/lpApr.ts

```diff
@@ -153,13 +153,13 @@ const getAprsForStableFarm = async (stableFarm: any): Promise<BigNumber> => {
   const stableSwapAddress = stableFarm?.stableSwapAddress
 
   try {
-    const dayAgo = sub(new Date(), { days: 1 })
+    const day7Ago = sub(new Date(), { days: 7 })
 
-    const dayAgoTimestamp = getUnixTime(dayAgo)
+    const day7AgoTimestamp = getUnixTime(day7Ago)
 
-    const blockDayAgo = await getBlockAtTimestamp(dayAgoTimestamp)
+    const blockDay7Ago = await getBlockAtTimestamp(day7AgoTimestamp)
 
-    const { virtualPriceAtLatestBlock, virtualPriceOneDayAgo } = await stableSwapClient.request(
+    const { virtualPriceAtLatestBlock, virtualPriceOneDayAgo: virtualPrice7DayAgo } = await stableSwapClient.request(
```

Comment:
Can we move this part to some kind of stable swap util file? So we can share code and don't need to implement the same logic in different places